snippets/csharp/programming-guide/classes-and-structs/object-collection-initializers/BasicObjectInitializers.cs

```diff
@@ -134,19 +134,19 @@ public class Cat
         public static void Main()
         {
             Cat cat = new Cat { Age = 10, Name = "Fluffy" };
-            Cat sameCat = new Cat("Fluffy"){ Age = 10 };
+            Cat sameCat = new Cat { Age = 10, Name = "Fluffy" };
```

Comment:
@jmarkman now, the updated line duplicates the line above. That looks strange.

I guess the intention was to show that one can mix using a constructor with the use of an object initializer. However, that is already covered in the earlier section in the same article:
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/object-and-collection-initializers#object-initializers

So, removing line at all with the `sameCat` variable seems to be a better solution. What do you think?

Reply:
Now that you've mentioned it, I would oppose that *and* my own submission. Instead, I'd propose that I should go back and simply change the `Cat` class to have a constructor with a name parameter. I was so focused on the fact that the line was erroring and that it was an initialization example that I didn't stop to think that it was mixing them together; couldn't see the forest for the trees.

Reply:
> ...change the `Cat` class to have a constructor with a name parameter.

Agree, it's another possible solution. Then, the parameter-less constructor must be explicitly added to the class as well.

Reply:
I've just added the parameterless and parameterized ctors to the `Cat` class and reverted my previous change to the hybrid constructor/initializer line.